mk_args.c.sh

```diff
@@ -75,6 +75,13 @@ write_struct() {
 
 	type_to_ctype "$t" "$snake_n"
     done
+    aditional=$(json-search -n additionalProperties <<< $st_info)
+
+    if [ "$aditional" != "null"  -a  "$aditional" != "false" ]; then
```

Comment:
`if [ "$additional" != "null" ] && [ "$additional" != "false" ]; then`

Reply:
why not, but why is it better ?

Reply:
annyway, I change the code to use bash built-in `[[`